packages/vats/src/proposals/kread-proposal.js

```diff
@@ -0,0 +1,24 @@
+import { E } from '@endo/far';
+
+/**
+ * @param {BootstrapPowers & {
+ *   consume: { kreadKit: any };
+ * }} powers
+ */
+export const repairKread = async ({ consume: { kreadKit: kreadKitP } }) => {
+  console.log('repairSubscribers');
+
+  const kreadKit = await kreadKitP;
+  const creatorFacet = kreadKit.creatorFacet;
```

Comment:
Consider destrucuring. Could be directly from the promise as 'kreadKit' is unused

Reply:
This is a direct checkout of the `release-mainnet1B` file